### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by FeliciousX**
updating these to latest will break the build

- cyclejs-utils@5

### CODE REVIEW

**Comment by FeliciousX** [comment]
~~updating cycle/dom breaks the example for rxjs :(~~ FIXED

File: examples/rxjs/src/index.ts [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import { Stream } from 'xstream';
-import { Observable } from 'rxjs';
+import { Observable, of as observableOf, pipe } from 'rxjs';
```
Comment (kiyui) [inline examples/rxjs/src/index.ts]: The `pipe` is not used here.
```suggestion
import { Observable, of as observableOf } from 'rxjs';
```

File: examples/rxjs/src/index.ts [hunk 2]
```diff
@@ -1,7 +1,9 @@
 import { Stream } from 'xstream';
```
Comment (kiyui) [inline examples/rxjs/src/index.ts]: This can be removed.

**Review by kiyui — CHANGES_REQUESTED** [review]
Minor changes required.

**Review by kiyui — APPROVED** [review]
:slightly_smiling_face:

**Comment by jvanbruegge** [comment]
Hi @FeliciousX, does this change still break the build? Or did I understand you wrong?

**Comment by FeliciousX** [comment]
everything works fine :D @jvanbruegge just had to update the example

**Comment by FeliciousX** [comment]
btw `browserify` is weird, the order of import matters

```typescript
import { run } from '@cycle/rxjs-run'
import { mapTo } from 'rxjs/operators'
```

must come first before importing rxjs operators.. otherwise it will cause a runtime error when we run the examples.

**Comment by FeliciousX** [comment]
i skipped updating `cyclejs-utils` but i updated rxjs to the latest version

**Comment by jvanbruegge** [comment]
Ok, i will merge an publish later this day, thank you

**Comment by FeliciousX** [comment]
thank you @jvanbruegge  :grinning:

**Comment by jvanbruegge** [comment]
Published as `v7.0.0`

**Pull request merged**